CONTRIBUTING.md

```diff
@@ -29,26 +29,53 @@ For each of the variants above, there are two sets of APIs to use for rendering
 
 ### Installation
 
-1. Clone the project down
-2. `cd` into the `js` folder from the top level of the project, and run `npm i` to install all dependencies
-3. Pull down the underlying `rive-runtime` submodule, which should be pointing to a specific commit.
+
+
+1. Install [Emscripten](https://emscripten.org/docs/getting_started/downloads.html). We build against `v3.1.43` in rive-wasm, so make sure to install and activate that version.
+2. Install [Premake5](https://premake.github.io/) and add it to your `PATH`.
+3. Install [Ninja](https://ninja-build.org/).
 
 ```sh
-# First time through if you just pulled down the project
+# Clone the Project
+git clone git@github.com:rive-app/rive-wasm.git
+
+# Install dependencies
+cd rive-wasm/js
+npm i
+
+cd ../
+
+# Pull down the underlying rive-runtime
 git submodule update --init --recursive
 
-# When updating the submodule after the initial pull
-git submodule update --recursive
+# When updating the submodule after the initial installation
+# git submodule update --recursive
+
+# Make sure the submodule is on the main branch
+cd wasm/submodules/rive-runtime/
+git checkout main
+
+cd ../../../js
```

Comment:
I vote that we instead just say:
- `cd` into the js dir.
- `cd` into `wasm/submodules/rive-runtime` and run..
Instead of hardcoding the commands like this.

Or alternatively do:
```
pushd wasm/submodules/rive-runtime/
....
popd
...
pushd js
```